hieunc-guestbook-dojo/static/js/guestbook/store/_GreetingStore.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Created by hieunc on 27/11/2015.
+ */
+
+define([
+	"dojo/_base/declare",
+	"dojo/_base/lang",
+	"dojo/Deferred",
+	"dojo/request/xhr"
+], function (declare, lang, Deferred, xhr) {
+	return declare("_GreetingStore", null, {
+		url: "/api/guestbook/default_guestbook/greeting/",
+		headers: {},
+		method: "get", //get,post,put,del
+		postData: {},
+		handleAs: "json",
+		expect: "json", //{json,httpStatus}
+
+		callBack: function (e) {
+			console.log()
+		},
+		errCallBack: function (err) {
+			alert("Failed to delete greeting !")
```

Comment:
- alert này có đúng ko? `errCallBack` đang dùng chung cho tất cả, vậy ko thể hiện thì alert là "Failed to delete greeting !" được. hãy dùng `console.log(err)`